app/models/hearing.rb

```diff
@@ -207,6 +207,13 @@ def scheduled_for_past?
     scheduled_for < DateTime.yesterday.in_time_zone(regional_office_timezone)
   end
 
+  # Checks the scheduled_datetime value and returns
+  # true if scheduled_datetime is not nil,
+  # else false
```

Comment:
```suggestion
  # Checks the scheduled_datetime value and returns
  # @return [Boolean] true if scheduled_datetime is not nil, else false
```
This is how you can formally notate a return type.